### DISCUSSION TITLE
Next Head is not populated from the server side

### DISCUSSION THREAD

**Discussion by geosigno** [vercel/next.js#24528] [Help]
I use the getServerSideProps method to fetch the data from the server. then I render my component along with a custom Head. 

While I have all the meta tags I created when I right click > inspect the page, there is no trace on the source code of the page. Which cause my share functionality to be obsolete, as no meta data are detected.

Is this the default behavior to not have a Head from the server?

**Comment by bobarros** [comment]
Are you respecting this rule when you try to render?

_'title', 'meta' or any other elements (e.g. script) need to be contained as direct children of the Head element, </br>or wrapped into maximum one level of <React.Fragment> or arrays—otherwise the tags won't be correctly picked up on client-side navigations._

It's from the documents ---> https://nextjs.org/docs/api-reference/next/head

I did this mistake myself not long ago.
